### ISSUE TITLE
delay in typing

### ISSUE THREAD

**Issue by Newbieyy1945** [giuspen/cherrytree#2121]
since a few weeks typing in cherrytree is delayed for some time or the cursor is following very slow. Typing in libreoffice-writer is instantaneous so the problem is in cherrytree I think

**Comment by giuspen** [comment]
Is it happening on every node? Can it be that your database is large, you are using password protected document and your autosave is on?

**Comment by Newbieyy1945** [comment]
Giuseppe Penone schreef op 2022-09-23 15:41:

> Is it happening on every node? Can it be that your database is large, 
> you are using password protected document and your autosave is on?
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>

Thanks for the response, after I filed the Issue I noticed that my 
database had doubled. All my mainlevel entries and all dependant entries 
had doubled. After I threw away all doubled entries the problem almost 
disappeared.

my autosave is on and I saved every minute. I changed that to 5 minutes. 
  So now I report a new problem: why did my database double? I remember I 
did a Save and Vacuum some time ago. I also moved to 0.99.49 
unintentionally, it just happened with an automatic update.

Regards

Marius Hentzen

Links:
------
[1] 
https://github.com/giuspen/cherrytree/issues/2121#issuecomment-1256230751
[2]
